src/main/java/com/github/luben/zstd/Zstd.java

```diff
@@ -1,11 +1,8 @@
 package com.github.luben.zstd;
 
-import java.nio.ByteBuffer;
-import java.util.Arrays;
-
 import com.github.luben.zstd.util.Native;
-import com.github.luben.zstd.ZstdCompressCtx;
-import com.github.luben.zstd.ZstdDecompressCtx;
```

Comment:
I am not sure why you can remove the `ZstdCompressCtx` and `ZstdDecompressCtx` - they are used in the code below

Reply:
Hmm, I see, it's the same package